mmedit/models/losses/perceptual_loss.py

```diff
@@ -119,7 +119,8 @@ class PerceptualLoss(nn.Module):
     """
 
     def __init__(self,
-                 layer_weights,
+                 layer_weights_perceptual,
+                 layer_weights_style,
```

Comment:
That would affect all models using perceptual loss, leading to a several compatibility issue. 

A workaround is to keep `layer_weights` for perceptual loss. We can then add `layer_weights_style`. By default it is None and when it is not None, we initialize `self.vgg_style`. The computations in `forward` are then based on whether `self.vgg_style` exists.